src/app/notifications/guards.ts

```diff
@@ -1,5 +1,12 @@
+import { getType } from 'typesafe-actions';
+import * as actions from './actions'
 import { appMessageType } from './reducer';
-import { AnyNotification, AppMessageNotification } from './types';
+
+import { AnyNotification, AppMessageNotification, QueuelessNotifications } from './types';
 
 export const isAppMessage = (message: AnyNotification): message is AppMessageNotification =>
   message.type === appMessageType;
+
+export const isQueuelessNotification =
+  (notification: AnyNotification): notification is QueuelessNotifications =>
+    notification.type === getType(actions.searchFailure)
```

Comment:
Not sure if the name `isSearchFailureNotification` would be better.
Probably in the future, there will be more notifications which should be displayed outside of the queue, but for everything with `queueless` was confusing for me

+ newline

Reply:
I'm not sure. I was expecting this kind of notification to not be one-time thing only. I will change it if needed.